### PR TITLE
Add a warning about GH colab too large to render on github preview.

### PR DESCRIPTION
**Pull request by tjann**

### CODE REVIEW

File: tutorials.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Python.
 
 -   **Case Study: COVID-19 Feature Exploration Analysis (by Google Health)**
 
-    [Google Colab](https://colab.research.google.com/drive/1LLteGjXifwSsD-YsGwBnI-i96G777Q7j){:target="_blank"}, [Github](https://github.com/datacommonsorg/api-python/blob/master/notebooks/COVID_19_Feature_Exploration_Analysis_with_Data_Commons.ipynb){:target="_blank"}
+    [Google Colab](https://colab.research.google.com/drive/1LLteGjXifwSsD-YsGwBnI-i96G777Q7j){:target="_blank"}, [Github (too large to render preview)](https://github.com/datacommonsorg/api-python/blob/master/notebooks/COVID_19_Feature_Exploration_Analysis_with_Data_Commons.ipynb){:target="_blank"}
```
Comment (pradh) [inline tutorials.md:24]: how does

   Github<sub>(too large to render preview)</sub>

look?

**Review by pradh — APPROVED** [review]

File: tutorials.md (hunk 1 above)
Comment (tjann) [reply tutorials.md]: Pretty good!

**Review by beets — COMMENTED** [review]
It actually just loaded fine for me on github:
image

If we want to keep that warning, I'd prefer using plain parenthesis (sub calls too much attention to it).

**Comment by tjann** [comment]
It just loaded for me too... maybe it was a temporary GitHub thing.

**Pull request closed without merging**